jiant/tasks/tasks.py

```diff
@@ -330,6 +334,14 @@ def process_split(
             split, indexers, model_preprocessing_interface, is_pair=False
         )
 
+    def update_metrics(self, out, batch):
+        logits = out["logits"]
+        labels = out["labels"]
+        tagmask = batch.get("tagmask", None)
```

Comment:
Looks like `tagmask` is defined but never used (same in a few more locations in this file). Delete?